TodoApi/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -29,7 +29,7 @@ public static WebApplicationBuilder AddOpenTelemetry(this WebApplicationBuilder
             builder.Logging.AddOpenTelemetry(logging =>
             {
                 logging.SetResourceBuilder(resourceBuilder)
-                       .AddOtlpExporter();
+                        .AddOtlpExporter(c => c.Endpoint = new Uri(otlpEndpoint));
```

Comment:
```suggestion
                       .AddOtlpExporter(c => c.Endpoint = new Uri(otlpEndpoint));
```

Reply:
Isn't this endpoint automagically used when `OTEL_EXPORTER_OTLP_ENDPOINT` is set?

Reply:
Yeah, you are right, just checked it. It was so, that the app wasn't exporting spans to the Jaeger and at first I tried to fix it, setting explicit the export-endpoint. 
I removed it, now it is just `AddOtlpExporter()`.